lib/geolookup/usa/city.rb

```diff
@@ -0,0 +1,17 @@
+module Geolookup
+  module USA
+    module City
+
+      CITY_MAP = 'CITIES.yml'
+
+      @cities_by_name
```

Comment:
what's the point of this line?

Reply:
i noticed that as part of the coding pattern, it's invoked so that it actually loads the yml file ahead of time when the class is loaded, see

https://github.com/Spokeo/geolookup/blob/master/lib/geolookup/usa/metro.rb

it has..
```
module Metro
      METRO_CODE_TO_NAME_FILE = 'METRO_CODE_TO_NAME.yml'

      @metro_code_to_name

      ###################################################################
      # self.code_to_name
      #
      # Given a metro code output the metro name
      # Else return nil
      #
      # EX: code_to_name(4) => "Abilene"
      def self.code_to_name(metro_code)
        @metro_code_to_name ||= Geolookup.load_hash_from_file(METRO_CODE_TO_NAME_FILE)
        get_value_from_hash(@metro_code_to_name, metro_code.to_s.to_i)
      end
```

Reply:
ah ok